### DISCUSSION TITLE
Debugging Rust Code with codelldb

### DISCUSSION THREAD

**Discussion by Antenna00** [helix-editor/helix#6355] [Q&A]
Hello.
I followed the following wiki and debugging seems to work fine but I'm encountering the issue that I'm not sure why it's happening.
https://github.com/helix-editor/helix/wiki/Debugger-Configurations#install-debuggers

1) When I attempt to start debugging from keybind "Space" -> "g" -> "l" and select the binary, "\target" folder where executable file is located doesn't seem to appear in suggestion. 
I've manually entered "\target\debug\XXX.exe" then it works.

2) When I start the debug mode I get some binary??? on the top of the console and I'm wondering what this is. 
![image](https://user-images.githubusercontent.com/85468264/226103466-0f4aa3b2-6603-4faf-9f8b-b3b717d61b36.png)

I wonder if anyone encounters the same issue.
Thank you.

**Comment by wildwestrom** [comment]
This is related to this issue, just for more info.
https://github.com/helix-editor/helix/issues/6481

**Reply by Antenna00** [reply]
Thanks a lot. I just noticed the answer.
Closing the discussion. Best :)
